ansible/vars.yml

```diff
@@ -8,7 +8,7 @@ postgres_major:
 
 # Full version strings for each major version
 postgres_release:
-  postgresorioledb-17: "17.0.1.041-orioledb"
+  postgresorioledb-17: "17.0.1.042-orioledb"
```

Comment:
@burmecia we'll need to please move these numbers up prior to merge, as there are already releases out with these numbers

Reply:
sure, I've changed it.